src/pages/patientView/PatientViewPage.tsx

```diff
@@ -47,6 +51,7 @@ interface IPatientViewState {
     cnaSegmentData: any;
     mutationData: any;
     hotspotsData?: IHotspotData;
+    cosmicData?: ICosmicData;
```

Comment:
We should try to distinguish between models (api data) and view-models, which are transformations necessary for view layer and which therefor don't necessarily need to be decoupled from views.   We should wait to really worry about this when mobx is in the picture.  Which should be next week.